core/app/modules/import-export/assets/js/pages/import/import-complete/import-complete.js

```diff
@@ -41,22 +42,50 @@ export default function ImportComplete() {
 			};
 		},
 		kitData = useMemo( () => getKitData(), [] );
-
-	useEffect( () => {
-		if ( ! uploadedData ) {
-			navigate( '/import' );
-		}
-	}, [] );
+		useEffect( () => {
+			if ( ! uploadedData ) {
+				navigate( '/import' );
+			} else if ( 'kit-library' === referrer ) {
```

Comment:
extract to some constant the will be shared everywhere

Reply:
How is it related to the previous condition?

Reply:
It can be in a different condition block, placed that this way as I wanted the condition to check if the `includedData` value is truthy. 
I have separated this into two different conditions.